app/src/test/java/com/x8bit/bitwarden/data/vault/repository/VaultRepositoryTest.kt

```diff
@@ -1243,6 +1243,100 @@ class VaultRepositoryTest {
             }
         }
 
+    @Test
+    @Suppress("MaxLineLength")
+    fun `unlockVaultWithAuthenticatorSyncKey with VaultLockManager Success should return Success`() =
```

Comment:
You need to update the test names now.

Reply:
🤦 thank you. 

71357ee2f2fc39c91a6b85a3d798cd7b50f21b85